support-frontend/assets/components/forms/standardFields/input.jsx

```diff
@@ -8,10 +8,12 @@ import './input.scss';
 
 // ----- Component ----- //
 
-function Input(props: {}) {
-  return <input className="component-input" {...props} />;
+function Input({ forwardRef, ...props }: {forwardRef?: ?Function}) {
```

Comment:
Explain `forwardRef` to me please

Reply:
[here](https://reactjs.org/docs/forwarding-refs.html). The main idea is to be able to access the underlying html element's ref higher up the component chain, but it's a bit hard t wrap your head around.

preact doesn't have the actual `forwardRef` api, and setting `ref` doesn't work (it returns the wrong html element) so I figured by naming it `forwardRef` the next developer could google it and land here